build/cdc/dev/.env.sample

```diff
@@ -38,5 +38,8 @@ CLOUDSQL_INSTANCE=
 # Use custom data commons environment.
 FLASK_ENV=custom
 
+# Optional: Google Analytics Tag ID
```

Comment:
Can we put the same comment in here as it is in `env.list`?

> (Optional) Set to your Google Tag ID to enable Google Analytics. See https://support.google.com/analytics/answer/9539598

Reply:
done